packages/owlbot-bootstrapper/package.json

```diff
@@ -23,6 +23,7 @@
     "lint": "cd cli && npm run lint && cd ../app && npm run lint && cd ../common-container && npm run lint"
   },
   "dependencies": {
+    "@google-automations/git-file-utils": "^1.2.1",
```

Comment:
Is this root package.json even necessary? app, cli, and common container all have their own separate package.json files.

Reply:
Not necessary.